src/domain/kaivuilmoitus/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { cloneDeep, omitBy } from 'lodash';
+import _, { cloneDeep, omitBy } from 'lodash';
```

Comment:
`isEqual` could be imported same way as `cloneDeep` and `omitBy` without the need to import the whole lodash package.

Reply:
Thank you for pointing this out! Fixed it now.